src/diagnostics/LogPointsManager.ts

```diff
@@ -158,7 +165,7 @@ export class LogPointsManager extends vscode.Disposable {
         const debugSessionManager = this._debugSessionManagerMapping[params.vscodeDebugSessionId];
 
         if (!debugSessionManager) {
-            vscode.window.showErrorMessage(`Cannot find debug session with id ${params.vscodeDebugSessionId}`);
+            vscode.window.showErrorMessage(`The debug session associated with this file has expired. Please close this file and open it again from LOADED SCRIPTS explorer of an active debug session.`);
```

Comment:
Please update the message to remove 'debug' from it. Say 'active logpoints session.' instead.